docs/tests/merge_sort.php

```diff
@@ -0,0 +1,41 @@
+<?php
+// #Metrics: {"complexity": 5, "nrMethods": 2, "nrClasses": 0}
+// #LineComplexity: 2
+function merge_sort($my_array)
```

Comment:
can we have a standard formatting?
here you are using tabs ...

Reply:
Yes, I fix it. Took these test files directly from the older version's unit tests.

Reply:
Done 👌